schemas/blocks/list_collector.json

```diff
@@ -42,12 +42,6 @@
       "for_list": {
         "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
       },
-      "add_answer": {
-        "$ref": "list_collector.json#/answer_reference"
-      },
-      "remove_answer": {
-        "$ref": "list_collector.json#/answer_reference"
-      },
```

Comment:
It looks like `list_collector.json#/answer_reference` is no longer used anywhere, if so the definition of it should also be removed.